src/main/resources/locales/en/blob-view-json.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
For completeness, I'd copy the JSON files into the welsh folder too just in case someone navigates to this page after selected welsh

Reply:
done